.github/workflows/CI.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
```

Comment:
any reason why we use v1 (since there is a stable v2 and we set this up initially for this repo, I feel we should use the latest stable version)?

Reply:
yeah, I followed the docs tbh: https://docs.github.com/en/actions/guides/building-and-testing-nodejs

I'll check whether we can use the `v2` instead. Thanks!